miniconf/tests/packed.rs

```diff
@@ -0,0 +1,71 @@
+#![cfg(feature = "json-core")]
+
+use miniconf::{Error, Packed, Tree, TreeKey, TreeSerialize};
+
+#[derive(Tree, Default)]
+struct Settings {
+    a: f32,
+    #[tree(depth = 1)]
+    b: [f32; 2],
+}
+
+#[test]
+fn packed() {
+    let mut p = String::new();
+
+    assert_eq!(
+        Settings::path(Packed::default(), &mut p, "/"),
+        Err(Error::TooShort(0))
+    );
+    p.clear();
+
+    for q in Settings::iter_paths::<String>("/") {
+        let q = q.unwrap();
+        let (a, _d) = Settings::packed(q.split("/").skip(1)).unwrap();
+        Settings::path(a, &mut p, "/").unwrap();
+        assert_eq!(p, q);
```

Comment:
Can you name them something more logical than `p`, `q`, `a`, and `d`? I don't know what these variables are all supposed to be.

Reply:
done